golink.go

```diff
@@ -169,17 +170,65 @@ func Run() error {
 	if err := srv.Start(); err != nil {
 		return err
 	}
+
 	localClient, _ = srv.LocalClient()
+out:
+	for {
+		ctx, cancel := context.WithTimeout(context.Background(), 10*time.Second)
+		defer cancel()
+		status, err := srv.Up(ctx)
+		if err == nil && status != nil {
```

Comment:
may want to log the error message

Reply:
Happy to add back in but I decided to omit the logging here to prevent the authentication link that we print when no `TS_AUTHKEY` is specified from being visually lost in the output.